Libraries/Image/RCTImageLoader.m

```diff
@@ -377,10 +371,24 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                    userInfo:nil], nil);
           return;
         }
-      }
 
-      // Call handler
-      completionHandler(nil, data);
+        NSString *responseDate = ((NSHTTPURLResponse *)response).allHeaderFields[@"Date"];
```

Comment:
@mmmulani Because we never know the response has cached without ```Date``` header, so I just ignore checking decoded cache and just let them decoded.
This case is so rare, so it could be ok to ignore caching, I guess.
I've just tested with simple NodeJS ExpressJS server with removing header (```res.removeHeader('Date')```), and I checked there are no Date header with Chrome Developer Tools, and with react-native's UIExplorer example, finally checked this logic work nicely.